.github/workflows/tag_image_push.yml

```diff
@@ -24,9 +24,27 @@ on:
           - OSS
           - EE
           - README
+      IS_LTS:
+        description: 'Is LTS release'
+        required: true
+        default: 'false'
+        type: choice
+        options:
```

Comment:
I had some problems with boolean types in github actions (probably related to https://github.com/actions/runner/issues/1483#issuecomment-969295757) It's safer to use string here

Reply:
not sure the issue you had but reading recent messages in https://github.com/actions/runner/issues/1483#issuecomment-969295757, looks like should not be an issue anymore

I forked and tested repo mentioned in https://github.com/actions/runner/issues/1483#issuecomment-2157791523
and boolean type seems to work as far as I can tell. Also tested `On push` with tag and seems to be working 

may be issue was with workflow_call or workflow_call or pr ones. Or may be i see comment on setting ENV is still an issue and using `fromJSON` resolves it

Can you check please? 

Don't won't to drag this but prefer we check as its we can learn something and avoid this in the future

Reply:
`IS_LTS` input is used to set environment variable which is string, I doesn't bring anything useful to the table.

Also, from what I see here, it still the case: https://github.com/actions/runner/issues/1483#issuecomment-2288233626

Reply:
Yes. I mentioned about ENV above and thats the comment I was thinking and looks like can be resolved using `fromJSON`
I will resolve but perhaps in the future lets try boolean first and see if that works

Reply:
I'm afraid it can be error prone, especially in if conditions, where nulls (and other values) are interpreted as a `false`, see https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/evaluate-expressions-in-workflows-and-actions#literals

> Note that in conditionals, falsy values (false, 0, -0, "", '', null) are coerced to false and truthy (true and other non-falsy values) are coerced to true.

Reply:
I think I saw that at some point but to me it made sense the way it works. I don't see as an issue but may be depends on the context or may be how it translate into BASH. 

I see [this ](https://github.com/actions/runner/issues/1483#issuecomment-2288233626)sets type as bool then converts it to string when setting ENV and then check using string everywhere. Others say set input as `string` and then use `fromJSON ` 

Must say all this is little messy :(